ingeniamotion/information.py

```diff
@@ -196,6 +197,20 @@ def get_node_id(self, alias: str = DEFAULT_SERVO) -> int:
             return int(drive.target)
         else:
             raise IMException("You need a CANopen communication to use this function")
+        
+    def get_baudrate(self, alias: str= DEFAULT_SERVO):
+        """Get the baudrate of a CANopen network.
```

Comment:
I think it may be: "Get the baudrate of target servo", because the function input is a servo.